packages/ensnode-utils/src/subname-helpers.ts

```diff
@@ -58,6 +58,18 @@ const UNINDEXABLE_LABEL_CHARACTER_CODES = new Set(
   UNINDEXABLE_LABEL_CHARACTERS.map((char) => char.charCodeAt(0)),
 );
 
+const UNKNOWN_LABEL_PREFIX = "[" as const;
+const UNKNOWN_LABEL_SUFFIX = "]" as const;
+
+type UnknownLabel = `${typeof UNKNOWN_LABEL_PREFIX}${string}${typeof UNKNOWN_LABEL_SUFFIX}`;
+
+export const isUnknownLabel = (label: string): label is UnknownLabel =>
+  label.startsWith(UNKNOWN_LABEL_PREFIX) &&
+  label.endsWith(UNKNOWN_LABEL_SUFFIX) &&
+  label.length === 66;
```

Comment:
@djstrong did you mean something like:
```suggestion
  label.startsWith(UNKNOWN_LABEL_PREFIX) &&
  label.endsWith(UNKNOWN_LABEL_SUFFIX) &&
  label.length === 66 &&
  isHex(unknownLabelAsHex(label));
```

Reply:
yes

Reply:
Done